### DISCUSSION TITLE
Changement emote xp de guilde

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#2236] [Nouvelles idées]
Comme dit dans le titre, changer l'émote ⭐ en 🌟 pour l'xp de guilde.
Et garder ⭐ pour l'xp personnel.

Exemple: 
![Screenshot_20231021_112157](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/96150225/09041e8b-94b2-444f-bd73-8850f0e0d170)

**Comment by symsym-1629** [comment]
déjà utilisé pour les objets légendaires

**Reply by romain22222** [reply]
l'autre emote est déjà utilisé pour les objets épiques, donc argument invalide :doge:

**Reply by symsym-1629** [reply]
effectivement

**Reply by BastLast** [reply]
c'est les objets légendaires tho

**Comment by BastLast** [comment]
je suis pas contre l'idée @Bombafox  je te laisse ouvrir une PR avec les changements si tu y tiens :)
